urihandler/handler.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, uris=[]):
         self.uris = uris
 
     def handle(self, uri, request):
+        params = ""
+        if "?" in uri:
+            uri, params = uri.rsplit("?", 1)
```

Comment:
is er een reden voor rsplit ipv gewone split? ik zou initieel denken dat dat de gewone split de betere is voor als er vraagtekens in de query zouden zitten (die normaal url-encoded moeten zijn, maar ja...)

Reply:
uhm neen ja beetje gewoontje van urls te splitten op "/" dan ist meestal rsplit, maar idd gwn zal beter zijn.